include/field.hxx

```diff
@@ -122,6 +128,7 @@ class Field {
   virtual const IndexRange region(REGION rgn) const = 0;
  protected:
   Mesh * fieldmesh;
+  Coordinates * fieldCoordinates;
```

Comment:
Should this be snake_case: `field_coordinates`? Or possibly `fieldcoordinates` to be like `fieldmesh`?